x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/use_primary_navigation.tsx

```diff
@@ -13,7 +13,7 @@ import type { PrimaryNavigationProps } from './types';
 import { usePrimaryNavigationItems } from './use_navigation_items';
 import { useIsGroupedNavigationEnabled } from '../helpers';
 import { SecuritySideNav } from '../security_side_nav';
-import { useTourContext } from '../../guided_onboarding';
+import { useTourContext } from '../../guided_onboarding_tour';
```

Comment:
I think the navigation hook doesn't need to check for the tour anymore. We added this in 8.4 because the tour steps were pointing to the items in the sidenav and we wanted to prevent the collapsing of the nav if a tour step is shown. That is not needed anymore I think.